spec/controllers/reports/general_reports_controller_spec.rb

```diff
@@ -249,7 +249,7 @@ def assert_report_params_init
 
   def assert_report_init(_label)
     expect(response).to be_success
-    expect(assigns(:total_quantity)).to be_instance_of Fixnum
+    expect(assigns(:total_quantity)).to be_instance_of Fixnum # rubocop:disable Lint/UnifiedInteger
```

Comment:
`@total_quantity` is not an `instance_of?(Integer)`, but it is a `kind_of?(Integer)`.Updating.